### PR TITLE
feat(copyVariableName): class extraction from Vue template / JSX

### PR DESCRIPTION
**Pull request by Ilanaya**

### CODE REVIEW

File: src/features/copyVariableName.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ export const registerCopyVariableName = () => {
         if (!activeEditor || activeEditor.viewColumn === undefined) return
         // also would make sense to support mutliple selections/cursors
         const lineText = activeEditor.document.lineAt(activeEditor.selection.end).text
-        const regexps = [/\s*(?:const(?: {)?|let(?: {)?|type|interface|import(?: {)?) ([\w\d]+)/, /\.([\w\d-]+?)\s*{/]
+        const regexps = [/\s*(?:const(?: {)?|let(?: {)?|type|interface|import(?: {)?) ([\w\d]+)/, /\s*(?:\.|(?:.*(?:class|className)=["']))([\w\d-]+?)\s*[{"']/]
```
Comment (zardoy) [inline src/features/copyVariableName.ts:10]:
```ts
/\s*(?:\.|(?:.*(?:class|className)=["']))([\w\d-]+?)\s*[{"']/
//                                                      ^ whats the purpose of this char?
```

**Review by zardoy — CHANGES_REQUESTED** [review]

**Comment by zardoy** [comment]
it just works

File: src/features/copyVariableName.ts (hunk 1 above)
Comment (Ilanaya) [reply src/features/copyVariableName.ts:10]: @zardoy  .contact-card__about {
// &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;^ ----

Comment (sa2urami) [reply src/features/copyVariableName.ts:10]: I just bought 18 nuggets wishing to relive every moment with you

Comment (sa2urami) [reply src/features/copyVariableName.ts:10]: And two cheese toppings

Comment (zardoy) [reply src/features/copyVariableName.ts:10]: @sa2urami  sa2uram

Comment (zardoy) [reply src/features/copyVariableName.ts:10]: @sa2urami zardo#21428 battle

Comment (sa2urami) [reply src/features/copyVariableName.ts:10]: I'm afraid typescipt is malignant tumour which disable ability to check tg

Comment (zardoy) [reply src/features/copyVariableName.ts:10]: @sa2urami  I can say the same about ur ds

**Pull request merged**